addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -155,7 +163,7 @@ var ActivityMenu = Widget.extend({
             type: 'ir.actions.act_window',
             name: data.model_name,
             res_model:  data.res_model,
-            views: [[false, 'kanban'], [false, 'list'], [false, 'form']],
+            views: this._getViewsList(data.res_model),
```

Comment:
Shouldn't we instead take the views available on the target model ? Otherwise, we will have to hardcode every single model to add their own available views.
@tde-banana-odoo : what do you think ?

Reply:
It has already been discussed. Look at the screenshot I sent you about that.